Octokit/Models/Response/RepositoryStar.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Represents additional information about a star (such as creation time)
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class RepositoryStar
+    {
+        public RepositoryStar() { }
```

Comment:
The main benefit for using `protected set` was around testability, but I'm not sure that's so important now in hindsight...